x-pack/plugins/security_solution/public/common/components/url_state/initialize_redux_by_url.tsx

```diff
@@ -45,7 +45,7 @@ export const dispatchSetInitialStateFromUrl = (
       const sourcererState = decodeRisonUrlState<SourcererScopePatterns>(newUrlStateString);
       if (sourcererState != null) {
         const activeScopes: SourcererScopeName[] = Object.keys(sourcererState).filter(
-          (key) => !(key === SourcererScopeName.default && pageName === SecurityPageName.detections)
+          (key) => !(key === SourcererScopeName.default && pageName === SecurityPageName.alerts)
```

Comment:
I think we need to add rules details.

Reply:
@angorayc made sure that we are good here. So I am good

Reply:
fixed in https://github.com/elastic/kibana/pull/103659